docs/docs/platforms/openshift/OPENSHIFT.md

```diff
@@ -0,0 +1,100 @@
+# Deploy To OpenShift
+
+!!! warning "Support Disclaimer"
+    Helm is a Kubernetes package manager that orchestrates the provisioning of applications onto existing Kubernetes infrastructure. The requirements for this infrastructure are described in [Prerequisites](../../userguide/PREREQUISITES.md). The Kubernetes cluster remains your responsibility; we do not provide direct support for Kubernetes or the underlying hardware it runs on.
+
+    If you have followed our documentation on how to configure the Helm charts, and you're using correctly created components, we will then provide support if you encounter an error with installation after running the `helm install` command.
+
+    Read more about [what we support and what we don’t](../../troubleshooting/SUPPORT_BOUNDARIES.md).
+
+!!! info "Official OpenShift Support"
+    **Starting from version 1.18, Atlassian Data Center Helm charts officially include support for OpenShift**.
+
+## :simple-redhatopenshift: Deploy with a Restricted and Nonroot Security Context
+
+Atlassian Data Center Helm charts are vendor and cloud agnostic, and create objects from standard APIs that [OpenShift](https://www.openshift.com/){.external} fully supports.
+
+However, the default OpenShift **restricted** Security Context Constrain ([SCC](https://docs.openshift.com/container-platform/4.14/authentication/managing-security-context-constraints.html){.external}) requires pods to be run with a UID, and SELinux context that are allocated to the namespace. It means it is not possible to deploy DC Helm charts with the default settings (with pre-defined `securityContext` and containers that run as root). See [FAQs](../OPENSHIFT_FAQ) for typical errors.
+
```

Comment:
I'd reduce the clutter of "It means it is not possible" to something that flows a bit more easily, like "It's therefore not possible". Feel free to ignore this though if you want to keep your voice.